packages/block-library/src/file/block.json

```diff
@@ -34,6 +34,13 @@
 			"type": "string",
 			"source": "html",
 			"selector": "a[download]"
+		},
+		"displayPreview": {
```

Comment:
I like this option and I believe we should probably extend it to various file types. Like QuickView for Gutenberg.

Reply:
Yes, it looks like previews for different file types are being discussed 👍🏻